src/DynamoCore/DynamoCore.csproj

```diff
@@ -128,7 +130,6 @@
       <NodeHelpFiles Include="$(SolutionDir)..\doc\distrib\NodeHelpFiles\**\*.*" />
       <OpenSourceLicenses Include="$(SolutionDir)..\doc\distrib\Open Source Licenses\**\*.*" />
       <LocalizedResources Include="$(SolutionDir)..\extern\Localized\**\*.*" />
-      <ExternAnalytics Include="$(SolutionDir)..\extern\Analytics.NET\*.*" />
```

Comment:
another one I missed, why is this removed? Should it have been removed as part of your other PR?